client-misk-jooq/src/test/kotlin/app/cash/backfila/client/misk/JooqMenuBackfillDbSetup.kt

```diff
@@ -0,0 +1,64 @@
+package app.cash.backfila.client.misk
+
+import app.cash.backfila.client.misk.jooq.gen.tables.references.MENU
+import app.cash.backfila.client.misk.setup.JooqTransacter
+import okhttp3.internal.toImmutableList
+
+object JooqMenuBackfillDbSetup {
+
+  fun createNoMatching(transacter: JooqTransacter): List<Long> {
+    transacter.transaction("") { session ->
+      repeat((0..4).count()) {
+        session.newRecord(MENU)
+          .apply {
+            this.name = "beef"
+          }.let {
+            it.store()
+            it.id!!
+          }
+      }
+    }
+    return emptyList()
+  }
+
+  fun createSome(transacter: JooqTransacter): List<Long> {
+    return transacter.transaction("") { session ->
```

Comment:
Nit:
Add a comment like the class name or "filing db"

Could save someone time in the future.